core/protocols/aws-json-protocol/src/main/java/software/amazon/awssdk/protocols/json/BaseAwsJsonProtocolFactory.java

```diff
@@ -175,6 +179,10 @@ protected StructuredJsonFactory getSdkFactory() {
      * can be overridden by subclasses to customize behavior.
      */
     protected Map<MarshallLocation, TimestampFormatTrait.Format> getDefaultTimestampFormats() {
+        return getMarshallLocationFormatMap();
```

Comment:
Is there any reason we made this change? Seems a bit code smell to me to have a method that just calls another method.